template/visual-diff/integration/visualdiff.spec.js

```diff
@@ -1,12 +1,14 @@
 describe('check the site for visual regressions', () => {
-  beforeEach(() => {
```

Comment:
We had a discussion on this, and prefer the older way of having a test for each page. So if you could revert the `before`/`after` to `beforeEach`/`afterEach`. Sorry. Totally my bad! ☺️

Reply:
reverted